.eslintrc

```diff
@@ -243,6 +243,13 @@
     "unused-imports/no-unused-imports-ts": "error",
     "@typescript-eslint/no-explicit-any": ["error"],
     "@typescript-eslint/semi": ["error"],
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "argsIgnorePattern": "^_",
+        "varsIgnorePattern": "^_"
```

Comment:
Can you please explain what these two options do?

Reply:
These options are used to ignore those variables and arguments which start with an underscore(_).